src/controllers/handlers.ts

```diff
@@ -148,6 +148,15 @@ export const estateMapPngRequestHandler = async (context: {
     const selected = Object.values(tiles).filter(
       (tile) => tile.estateId && tile.estateId === estateId
     )
+    if (selected.length === 0) {
+      const headers: Record<string, string> = {
+        location: 'https://ui.decentraland.org/dissolved_estate.png',
+      }
+      return {
+        status: 302,
```

Comment:
I changed this to a 302 to cover the (unlikely) case of a dissolved estate being resurrected. It's not possible via de UI, but it is according to the contract (you can always transfer lands back into the estate even after dissolving it).